src/services/teams/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Wofür wird das hier global gebraucht? In den allermeisten Fällen sollte es doch lokal reichen oder durch refactoring entfallen.

Reply:
weil es da ganz viele _ in dem file gibt.

Reply:
So viele _ sind doch garnicht genutzt. Du kannst das Problem doch einfach beheben indem du die _ an den etwa 20 Stellen entfernst

Reply:
Das macht lint --fix später von alleine. @schurigh